src/guides/v2.4/config-guide/cache/two-level-cache.md

```diff
@@ -1,11 +1,12 @@
 ---
 group: configuration-guide
-title: L2 caching in the Magento application
+title: L2 (Level 2) caching in the Magento application
 functional_areas:
   - Configuration
   - System
   - Cache
 ---
+Level 2 cache is responsible for delivering to Level 1 cache such as memory cache.
```

Comment:
I think this needs a bit more explanation. Readers may ask "delivering what to L1?"